internal/db/profiles_test.go

```diff
@@ -180,6 +181,61 @@ func upsertEvalStatus(
 	require.NoError(t, err)
 }
 
+func createRuleEntity(
```

Comment:
All the changes in this test file are needed to use the evaluation history tables instead of the old rule evaluations tables when testing profile status transitions.